docs/source/grid.rst

```diff
@@ -0,0 +1,88 @@
+Lucid Grid Configuration
+========================
+
+``lucid``'s main grid area is an auto-generated series of buttons and indicators,
+where each indicator represents a device
+loaded from a ``happi`` database.
+
+.. figure:: /_static/lucid_grid.png
+   :scale: 50 %
+   :alt: Lucid display with the grid highlighted.
+
+
+Which Devices are Included?
+---------------------------
+
+When ``lucid`` is invoked, the only required argument is the positional ``beamline`` argument.
+This argument is used as a ``happi`` search term to fill the grid.
+
+This is equivalent to the following search using the ``happi`` command line,
+assuming your beamline is named BEAMLINE:
+
+.. code-block:: bash
+
+    happi search beamline=BEAMLINE active=True
+
+
+All devices that match this search and load without errors will be included in the grid.
+All other devices will be excluded.
+The search is case-sensitive.
+
+Note that inactive devices are not included,
+and that this only targets happi entries that have a beamline associated with them at all.
+This requires the ``happi`` containers defined in ``pcdsdevices``,
+which define the ``beamline`` field.
+
+Note: if you prefer not to load any devices, you can skip this step by passing
+a command-line parameter:
+
+.. code-block:: bash
+
+    lucid --skip_happi MY_BEAMLINE
+
+
+How are the Devices Arranged in the Grid?
+-----------------------------------------
+
+The grid is defined by the various metadata values found in the ``happi`` database.
+The default metadata keys in use are:
+
+- ``location_group``, to select which column to put a device into
+- ``functional_group``, to select with row to put a device into
+
+That is to say, by default we expect the row headers on the left to be functional names like
+"imager" or "vacuum" and column headers on the top to be location names like "SB3" or "VLS".
```

Comment:
```suggestion
"imager" or "vacuum" and column headers on the top to be location names like "Section 01" or "Section 02".
```
I like the docs a lot better with pictures, one last nitpick might be to make the examples match the images. 

We could take this a step further with the buttons config, but if we do I'd advocate showing two configs.  One with the descriptive values and one that matches the toolbar shown in the image.

Reply:
I think I'm not motivated to make an example config that matches the image, I'd be better off redoing the images to be the current config and pasting the current KFE config in.

Instead, I'll add a link to the lucid configs backup repo (which I'll update again at some point) so at least there's a way to see a bunch of valid configs.

I'm also going to merge your suggestion here